modules/cross-account-member/main.tf

```diff
@@ -14,9 +19,8 @@ module "account" {
 # Send invite from administrator account
 module "member" {
   source = "../member"
-  providers = {
-    aws = aws.administrator
```

Comment:
I don't want to switch this around. The default `aws`  provider should be the invitee. The administrator should be a custom aliased provider.

Reply:
Switched back.